controllers/auth.js

```diff
@@ -33,11 +33,12 @@ const logout = async (req, res) => {
   res.status(200).json(logoutInfo)
 }
 
-const activate = async (req, res) => {
-  const activationLink = req.params.link
-  await authService.activate(activationLink)
+const confirmEmail = async (req, res) => {
+  const confirmToken = req.params.token
```

Comment:
just `const {token}= req.params`

Reply:
thanks, but my way is perfectly fine as well :)